docs/language/crypto.md

```diff
@@ -33,6 +33,9 @@ pub enum SignatureAlgorithm: UInt8 {
 
     /// ECDSA_Secp256k1 is Elliptic Curve Digital Signature Algorithm (ECDSA) on the secp256k1 curve.
     pub case ECDSA_Secp256k1 = 2
+
+    /// BLS_BLS12381 is BLS signature scheme on the BLS12381 curve.
```

Comment:
Just to use the original [name](https://electriccoin.co/blog/new-snark-curve/) of the curve. 
```suggestion
    /// BLS_BLS12381 is BLS signature scheme on the BLS12-381 curve.
```

Reply:
You might be asking why I used `BLS_BLS12381` and not `BLS_BLS12_381`:
I used the underscore in `ECDSA` to separate the scheme name and the parameter name (here the curve), so I didn't want to add an extra underscore in the curve name `BLS12_381`. I'm open to better suggestions!

Reply:
Oh, thanks for the explanation! In that case I'd recommend `BLS_BLS12_381`, because it's more readable IMHO

Reply:
That works for me!

Reply:
In `KMAC128_BLS_BLS12381` hashing algo, we use `BLS_BLS12381`. Should we use the same pattern here as well (or use `BLS_BLS12_381` there - but that would make the name really long)?

I'm not sure whether they both refer to the same thing though.

Reply:
They do refer to the same thing, it's better to be consistent and have `BLS_12_381` for both 🤷‍♂️

Reply:
Updated to use `BLS_BLS12_381` format for the signature algo in 1e25c63.
I will send a separate PR updating the hash algorithm to use the same.